lang/guile/Portfile

```diff
@@ -3,7 +3,8 @@
 PortSystem 1.0
 
 name                guile
-version             2.0.14
+version             2.2.0
+revision            1
```

Comment:
This may be an error.  I recall, for another port, to add a revision number.  Perhaps it is zero-indexed, and if the first (0th) revision, then this is not necessary.